src/Microsoft.AspNet.Server.Kestrel/Http/FrameOfT.cs

```diff
@@ -111,7 +112,7 @@ public class Frame<TContext> : Frame
                             _application.DisposeContext(context, _applicationException);
 
                             // If _requestAbort is set, the connection has already been closed.
-                            if (!_requestAborted)
+                            if (Volatile.Read(ref _requestAborted) == 0)
```

Comment:
You know someone, somewhere, is going to run a Kestrel websocket server on an arm Windows phone as part of their UWP app, and curse the memory model - just saying... :wink: